src/__test__/Home.test.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import Home from '../components/Home';
+import Login from '../components/login';
 
 
 describe('App', () => {
   it('should match snapshot', () => {
     const wrapper = shallow(<Home />);
     expect(wrapper.find('h1').text()).toBe('Welcome to Authors Haven');
   });
+  it('should match snapshot', () => {
+    const wrapper = shallow(<Login />);
+    expect(wrapper.find('h1').text()).toBe('login here');
```

Comment:
Use this for snapshot tests `expect(wrapper).toMatchSnapshot();`

Reply:
I have implemented this feedback